### PR TITLE
Sir 788 ecs tasks

### PR DESCRIPTION
**Pull request by matchbookmac**
### Why is this pull request necessary?
Want to start using ECR, ECS, and Parameter Store in deploying applications.

 ### How does it fix the issue?
- Adds commands to push/pull env vars to parameter store.
- Adds commands to build and tag docker images, then push and image to ECR.

 ### What side effects might this have?

### CODE REVIEW

File: bin/pull_env [hunk 1]
```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
```
Comment (matchbookmac) [inline bin/pull_env:1]: Can refactor these bash scripts into ruby later if needed, for now I know they work.

**Review by jasonpilz — DISMISSED** [review]
🙏

File: bin/pull_env (hunk 1 above)
Comment (jasonpilz) [reply bin/pull_env:1]: Bash good with me 👍

**Review by jasonpilz — APPROVED** [review]
⬆️

**Pull request merged**